### DISCUSSION TITLE
win10+TeXLive2023+TeXstudio4.7.2环境下，使用最新SJTUThesis基线编译出错

### DISCUSSION THREAD

**Discussion by erwincv** [sjtug/SJTUThesis#1014] [提问] [answered]
使用环境：
Windows 10 家庭中文版  22H2；tlmgr revision 69686 (2024-02-03 08:34:54 +0100)
TeX Live (https://tug.org/texlive) version 2023；TeXstudio 4.7.2 (git 4.7.2)
配置信息：
TeXstudio已设置为TeXLive为默认编译器。之后，按照文件“README.md”中，对TeXstudio的设置要求：
“Commands-Latexmk一项修改为`latexmk -silent -synctex=1 -xelatex %`

Build-Default Compiler一项修改为`txs:///latexmk`”
已做更改。如下：
![image](https://github.com/sjtug/SJTUThesis/assets/58477556/d80ee9eb-23b6-4f62-94ab-ab49806ea4c7)
![image](https://github.com/sjtug/SJTUThesis/assets/58477556/97216912-59c0-4240-ae51-f3b62ae26989)

**选择“main.tex"至高亮后，build错误信息如下：**”
开始 : latexmk.exe -silent -synctex=1 -xelatex"main"

Latexmk: -xelatexmain unknown option
D:\Program Files\texlive\2023\bin\windows\runscript.tlu:921: command failed with exit code 10: perl.exe "d:\Program Files\texlive\2023\texmf-dist\scripts\latexmk\latexmk.pl" -silent -synctex=1 -xelatexmain
出现错误

开始 : "D:/Program Files/texlive/2023/bin/windows/bibtex.exe" "main"

I couldn't open file name `main.aux'

出现错误

开始 : latexmk.exe -silent -synctex=1 -xelatex"main"

Latexmk: -xelatexmain unknown option
D:\Program Files\texlive\2023\bin\windows\runscript.tlu:921: command failed with exit code 10: perl.exe "d:\Program Files\texlive\2023\texmf-dist\scripts\latexmk\latexmk.pl" -silent -synctex=1 -xelatexmain
出现错误

开始 : latexmk.exe -silent -synctex=1 -xelatex"main"

Latexmk: -xelatexmain unknown option
D:\Program Files\texlive\2023\bin\windows\runscript.tlu:921: command failed with exit code 10: perl.exe "d:\Program Files\texlive\2023\texmf-dist\scripts\latexmk\latexmk.pl" -silent -synctex=1 -xelatexmain
出现错误
“
似乎是编译环境设置错误，困于我是小白，终究无所适从，恳请大佬不吝指点。多谢。

**Comment by erwincv** [comment]
错误显示：
![image](https://github.com/sjtug/SJTUThesis/assets/58477556/d6cd5391-e947-4089-947f-661fb92c8ece)

**Comment by AlexaraWu** [comment, marked as answer]
`latexmk` 配置 `%` 前面少了一个空格。

**Reply by erwincv** [reply]
多谢，确实是这个原因。
